css_tasks/task11/js/scroll.js

```diff
@@ -0,0 +1,38 @@
+import {elements} from './elements.js';
+export function scroll() {
+  elements.then((data) => {
+    const {navbar, header, links} = data;
+
+    // smooth scrolling
+    const handleScroll = (item, e) => {
```

Comment:
handleScroll but all on click looks like something went wrong

why not just pass event and then get target from event?